config/mainnet.go

```diff
@@ -156,10 +156,12 @@ func MainnetConfig() Config {
 			BeaconSyncWeightUnits:    800,
 		},
 		POET: activation.PoetConfig{
-			PhaseShift:        240 * time.Hour,
-			CycleGap:          12 * time.Hour,
-			GracePeriod:       1 * time.Hour,
-			RequestTimeout:    1100 * time.Second, // RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
+			PhaseShift:                     240 * time.Hour,
+			CycleGap:                       12 * time.Hour,
+			GracePeriod:                    1 * time.Hour,
+			PositioningATXSelectionTimeout: 50 * time.Minute,
+			// RequestTimeout = RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
```

Comment:
if this is not used anymore, it can probably be removed

Reply:
It is still used in `submitPoetChallenge` of the `NIPostBuilder`?

Reply:
It is still used. In `submitPoetChallenge` in the `NIPostBuilder`.

Reply:
Oh... I didn't realize it explains the calculation :) It looks like an old variable definition at first sight.